src/worker/lambda/packages/packagePuller.go

```diff
@@ -175,6 +179,29 @@ func (pp *PackagePuller) sandboxInstall(p *Package) (err error) {
 			return err
 		}
 	}
+	//gets the size of the current package
+	cmd := exec.Command("du", "-sb", scratchDir)
+	var out bytes.Buffer
+	cmd.Stdout = &out
+	err = cmd.Run()
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Fatal will kill the server.  I don't think that's a proportionate response to an error.  Better to return it back.